src/sentry/scim/endpoints/members.py

```diff
@@ -370,16 +373,29 @@ def post(self, request: Request, organization) -> Response:
         - Sentry's SCIM API doesn't currently support setting users to inactive,
         and the member will be deleted if active is set to `false`.
         - The API also does not support setting secondary emails.
+        - you can provide a sentryOrgRole to provision alongside the new user
         """
 
+        if (
+            features.has("organizations:scim-orgmember-roles", organization, actor=None)
+            and "sentryOrgRole" in request.data
+        ):
+            role = request.data.get("sentryOrgRole").lower()
```

Comment:
do you have to do `.get` if you've already validated `sentryOrgRole` is in `data`?

Reply:
nope! force of habit.